### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by MattGomb**
Dear Reviewer!

In this PR, I connected my project to the provided API with POST and DELETE.
I used an updated thunk syntax, but everything follows the required logic.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -38,29 +21,76 @@ export default function booksReducer(state = defaultState, action) {
...
       return state;
   }
 }
 
-// action creators for add and remove books
+// action creators for fetch, add and remove books
+
+export const fetchBook = (list) => ({ type: FETCH, list });
+
+export const fetchBookAsync = () => async (dispatch) => {
```
Comment (Ismailco) [inline src/redux/books/books.js]: - [ ] This functionality should be created using the `createAsyncThunk`, kindly consider making the changes so your thunk action creators use `createAsyncThunk` instead

File: src/redux/books/books.js [hunk 2]
```diff
@@ -38,29 +21,76 @@ export default function booksReducer(state = defaultState, action) {
...
+  dispatch(addBook(newBook));
+};
+
+export const removeBook = (id) => ({
   type: REMOVE,
-  id: payload.id,
+  id,
 });
+
+export const bookRemove = ({ id }) => async (dispatch) => {
```
Comment (Ismailco) [inline src/redux/books/books.js]: - [ ] Same here, kindly consider making the changes so your thunk action creators use `createAsyncThunk` instead

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @MattGomb, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (MattGomb) [reply src/redux/books/books.js]: @Ismailco Hi! The function uses an async thunk (a function that returns another function), i am just not using the words createAsyncThunk.

Can you consider this version to be okay?

File: src/redux/books/books.js (hunk 2 above)
Comment (MattGomb) [reply src/redux/books/books.js]: @Ismailco Same here as above :)

File: src/redux/books/books.js (hunk 1 above)
Comment (Ismailco) [reply src/redux/books/books.js]: @MattGomb, yes I know but the requirement specifically mentioned using the `createAsyncThunk`, kindly make sure your code uses this method to create the thunk action creator

**Review by rustam-tolipov — APPROVED** [review]
Hi @MattGomb, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MattGomb** [comment]
@Rustamxon7 Thank you for taking the time!

**Pull request merged**